docs/api-docs/channels/building-channel-apps.md

```diff
@@ -38,35 +52,35 @@ Here's a few things you'll need to create channel apps.
 
 ### Extend an existing app
 
-If you would like to integrate [Channels Toolkit](https://developer.bigcommerce.com/api-docs/channels/guide/channels-toolkit-reference) in to an existing app, see [Extending Existing Apps](https://developer.bigcommerce.com/api-docs/channels/guide/extending-existing-apps) or [Building Storefront Channels](https://developer.bigcommerce.com/api-docs/channels/tutorials/storefront) for detailed instructions.
+If you would like to integrate [Channels Toolkit](https://developer.bigcommerce.com/api-docs/channels/guide/channels-toolkit-reference) into an existing app, see [Extending Existing Apps](https://developer.bigcommerce.com/api-docs/channels/guide/extending-existing-apps) or [Building Storefront Channels](https://developer.bigcommerce.com/api-docs/channels/tutorials/storefront) for detailed instructions.
 
 ### Developing a new app
 
-If would like to start development on a new app that integrates channels toolkit, and this is your first time developing a BigCommerce app, see [Building Apps](https://developer.bigcommerce.com/api-docs/apps/guide/intro) for an in-depth BigCommerce app development guide.
+If you would like to start development on a new app that integrates channels toolkit, and this is your first time developing a BigCommerce app, see [Building Apps](https://developer.bigcommerce.com/api-docs/apps/guide/intro) for an in-depth BigCommerce app development guide.
 
 ## Gathering materials
 
 The BigCommerce APIs, webhooks, and UI components needed to build a channel app vary depending on the [type of channel](https://developer.bigcommerce.com/api-docs/channels/guide/overview#types-of-channels), functionality, and where the app is marketed (Channel Manager vs. App Marketplace).
 
-### APIs & Webhooks
+### APIs and webhooks
 
-The APIs that you integrate with and how you use them (read-only vs. modify) will determine what BigCommerce scopes your app requests when it is installed by a merchant and what endpoints you'll need to integrate with.
+The APIs that you integrate with and how you use them (read-only vs. modify) will determine what BigCommerce scopes your app requests when installed by a merchant and what endpoints you'll need to integrate with.
```

Comment:
Should it be "**how** BigCommerce scopes your app" instead of "**what** BigCommerce scopes your app"?

Reply:
In this case, I would use "what" since it's referring to the noun "OAuth scopes": https://oauth.net/2/scope/.